test/amoc_api_scenario_handler_SUITE.erl

```diff
@@ -51,6 +57,7 @@ end_per_testcase(_, _Config) ->
 
 get_scenario_status_returns_200_when_scenario_exists(_Config) ->
     %% given
+    given_amoc_controller_prepared(true),
```

Comment:
Literally passing atom `true` here is a bad style. When you look at this code you have no idea what it means.